### DISCUSSION TITLE
Add the management of OCG layers ?

### DISCUSSION THREAD

**Discussion by AndreC75** [sumatrapdfreader/sumatrapdf#2417] [Ideas]
Is it possible to add to Sumatra the management of OCG layers that make PDFs interactive as shown in this example below?

[demo-ocg](http://mirrors.ctan.org/macros/latex/contrib/ocgx/demo-ocgx.pdf)

**Comment by GitHubRulesOK** [comment]
Since layers are only applied at base level defaults (without any controls in mupdf-gl let alone custom X-object ones that are designed for acrobat pro) and thus only designed for equally massive 596 MB minimum 1,897 Files, 219 Folders Foxit as a windows alternative, However I was pleased to see Tracker Xchange at 206 MB across 653 Files, 20 Folders was able to change page appearances exceedingly well and fast.

Seriously its a supplemental feature to the ethos of keeping a cross platform document that should appear the same on any device such as a printer and thus printing controlled layers is always an issue, as it goes against the original pdf principles.
